### DISCUSSION TITLE
Sharing whole feedset, read-only, make it look like FreshRSS interface

### DISCUSSION THREAD

**Discussion by andrey-utkin** [FreshRSS/FreshRSS#6645] [Q&A] [answered]
1. There is a feature to make the default user's feeds visible read-only, e.g. https://news.decent.im/i/?a=normal&state=3 .
2. There is a (discouraged) way to share user's whole set of feeds using the configured auth token, e.g. `https://news.decent.im/i/?a=global&user=username&token=token1` . I am not even sure if it's a read-only view, or there's a possibility for a person having the hold of this token to modify something on user's behalf.
3. There is the recommended way to share, via a "user query", it's more flexible, but the view is different, and there doesn't seem to be a way to choose the view similar to FreshRSS interface: https://news.decent.im/api/query.php?user=freshrss&t=6Px7phMLuRKlmb1dhazyYf&f=html

So is there a way to modify the look of user query HTML link to look like №1 and №2?

**Comment by Alkarex** [comment, marked as answer]
The token approach is read-only, but gives the possibility to see everything from that user account (all feeds, all categories, etc.).
For number 3, not at the moment, but PR welcome.
